frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js

```diff
@@ -35,6 +35,17 @@ describe("scenarios > embedding > native questions", () => {
       cy.contains("39.58");
 
       filterWidget().should("not.exist");
+
+      // Test embedding with locale metabase#22561
+      cy.url().then(url => {
+        cy.visit({
+          url,
+          qs: {
+            locale: "de",
```

Comment:
One interesting note is that. When there's no translation file built. Only locales with 2 characters format would work. e.g. `de` but not `de-DE`

However, when there are translation files built, both format works i.e. both `de` and `de-DE` works.

I didn't dig much into why there is such discrepancy in the behavior.